forecasttools/pull_data_cdc_gov.py

```diff
@@ -0,0 +1,145 @@
+from datetime import date
+
+import polars as pl
+from cfasodapy import Query
+
+# Dataset IDs and metadata for commonly
+# used (within STF) data.cdc.gov datasets
+data_cdc_gov_datasets = pl.DataFrame(
+    {
+        "key": ["nhsn_hrd_prelim", "nhsn_hrd_final", "nssp_prop_ed_visits"],
+        "id": ["mpgq-jmmr", "ua7e-t2fy", "rdmq-nq56"],
+        "date_column": ["weekendingdate", "weekendingdate", "week_end"],
+        "location_column": ["jurisdiction", "jurisdiction", "county"],
+    }
+)
+
+
+def get_dataset_info(dataset_key: str) -> dict:
+    """Look up dataset information by key."""
+    filtered = data_cdc_gov_datasets.filter(pl.col("key") == dataset_key)
+    if filtered.height == 0:
+        raise ValueError(
+            f"Dataset key '{dataset_key}' not found in dataset table"
+        )
+    return filtered.to_dicts()[0]
+
+
+def _parse_comma_separated_values(cs_string: str) -> list[str]:
+    """Parse a comma-separated string into a list of stripped strings."""
+    return [value.strip() for value in cs_string.split(",")]
+
+
+def get_data_cdc_gov_dataset(
+    dataset_key: str,
+    start_date: str | date = None,
+    end_date: str | date = None,
+    additional_col_names: str | list[str] = None,
```

Comment:
This type annotation is incorrect. You expect a single string (though it can be comma-separated). I would prefer if the function works as the type annotation suggests.